branchtesting/src/main/java/branchexample/App.java

```diff
@@ -10,7 +10,8 @@ public void aMethod( String[] args )
     		// TODO fix this too..
     		int j = 5 / i;
     	}
-    	System.out.println("IN AMETHOD"); 
+    	System.out.println("IN AMETHOD");
+    	String password="XXXX";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this useless assignment to local variable "password". ([squid:S1854](https://sonarcloud.io/organizations/brian-cipollone-sonarsource-gith/rules#rule_key=squid%3AS1854))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.example.bcipollone%3Abranchtesting&pullRequest=17&issues=AWXs3dPBxJR_uy9B9KL_&open=AWXs3dPBxJR_uy9B9KL_)

[ISSUE_KEY]: AWXs3dPBxJR_uy9B9KL_